src/Components/Bodies/CodeBodyUnversionized.tsx

```diff
@@ -134,117 +118,146 @@ class CodeBodyUnversionized extends Component<Props, ICode> {
+        if (attributeValue.length) {
+            attributesHtml.push(
+                <div key={ind}>
+                    <h5>{translateJson["LBL_" + attribute.toUpperCase()]}</h5>
+                    <ul>
+                        {attributeValue.map((val, j) => (
+                            <li key={attribute + "_" + j}><p dangerouslySetInnerHTML={{__html: val}}/></li>
+                        ))}
+                    </ul>
+                </div>
+            )
         }
-        else return code;
     }
 
     /**
      * Render the CodeBodyUnversionized component
      * @returns {JSX.Element}
      */
     render() {
-        let translateJson = findJsonService(this.props.params.language)
-        let attributes_html = []
-        let parentBreadCrumbs = []
-
-        // TODO: below if else will be refactored into more compact code
-        // TODO: Collecting Breadcrumbs will be refactored into Utils since we can use it for both un- & versionized codes.
-        if(this.state.parents && this.state.parents.length > 0){
-            for(let i=this.state.parents.length-1; i>=0; i--){
-                parentBreadCrumbs.push(<Breadcrumb.Item
-                    key={i}
-                    onClick={() => this.goToChild(this.state.parents[i])}
-                    className="breadLink"
-                >{this.extractLabel(this.state.parents[i].code)}</Breadcrumb.Item>)
+        // Generate BreadCrumbs.
+        let parentBreadCrumbs = [];
+        if (this.state.parents) {
+            for (let i = this.state.parents.length - 1; i >= 0; i--) {
+                parentBreadCrumbs.push(
+                    <Breadcrumb.Item
+                        key={i}
+                        onClick={() => this.goToCode(this.state.parents[i])}
+                        className="breadLink"
+                    >{this.extractLabel(this.state.parents[i].code)}</Breadcrumb.Item>)
             }
         }
+
+        let translateJson = findJsonService(this.props.params.language);
+        let attributesHtml = [];
         let i = 1;
-        for(let attribute in this.state.attributes) {
-            if(this.state.attributes[attribute] !== null && this.state.attributes[attribute] !== undefined) {
-                if(this.state.attributes[attribute].length > 0 && attribute === "limitation") {
-                    attributes_html.push (
+
+        // Add all non null/empty/undefined code attributes to attributesHtml.
+        for (let attribute in this.state.attributes) {
+            // Skip attributes we do not want to show, like terminal, version, ... or add later (siblings and children).
+            if (skippableAttributes.includes(attribute)) { continue; }
+            // Get value of current attribute.
+            let attributeValue = this.state.attributes[attribute];
+            // Only show attribute if defined, not null or not empty.
+            if (!["", null, undefined].includes(attributeValue)) {
```

Comment:
Maybe also more elegant with filter and map functions.

Reply:
In general: more functional

Reply:
First: filter out all sublists and give them a name, then process all sublists using each or map. this should also be more readable.